<ui version="4.0" >
 <class>ConsoleChannel</class>
 <widget class="QWidget" name="ConsoleChannel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>35</width>
    <height>294</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>35</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>114</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>-1</number>
   </property>
   <property name="leftMargin" >
    <number>5</number>
   </property>
   <property name="topMargin" >
    <number>5</number>
   </property>
   <property name="rightMargin" >
    <number>5</number>
   </property>
   <property name="bottomMargin" >
    <number>5</number>
   </property>
   <item>
    <widget class="QToolButton" name="m_presetButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_valueLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>000</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="m_valueSlider" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="m_statusButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_numberLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>000</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
